site/pages/docs/index-en.hbs

```diff
@@ -38,3 +38,8 @@
 	<li>Follow the <a href="ref/docstmpl-en.html">documentation example template</a>.</li>
 	<li>At least one working example should be provided for each WET feature, theme or component. Additional examples should be provided for each parameter or option.</li>
 </ul>
+
+<section>
+	<h2 id="wet31">Notice about WET v3.1</h2>
+	<p>As of September 23, 2014, version 3.1 of the Web Experience Toolkit is no longer supported. The source code and documentation have been moved to the <a href="{{site.legacyrepolink}}">{{site.legacyreponame}}</a>repository.</p>
+</section>
```

Comment:
Add a blank line to the end of the file (red icon above indicates it is missing).